tests/h/views/api_groups_test.py

```diff
@@ -100,12 +100,74 @@ def open_groups(self, factories):
         return [factories.OpenGroup(), factories.OpenGroup()]
 
     @pytest.fixture
-    def anonymous_request(self, pyramid_request):
-        pyramid_request.user = None
+    def authenticated_request(self, pyramid_request, factories):
+        pyramid_request.user = factories.User()
         return pyramid_request
 
+
+@pytest.mark.usefixtures('CreateGroupAPISchema',
+                         'group_service',
+                         'GroupContext',
+                         'GroupJSONPresenter')
+class TestCreateGroup(object):
+
+    def test_it_inits_group_create_schema(self, pyramid_request, CreateGroupAPISchema):  # noqa: N803
+        views.create(pyramid_request)
+
+        CreateGroupAPISchema.assert_called_once()
```

Comment:
I think you want `assert_called_once_with()` (since it should be called with no args)